spec/models/ability_spec.rb

```diff
@@ -171,6 +171,35 @@
         end
       end
 
+      describe "just orga members, team\'s supervisor and team\'s students should be able to see offered conference for a team" do
+        let(:user) { FactoryGirl.create(:student)}
+
+        context 'if it is an student of another team' do
+          it { expect(ability).not_to be_able_to(:see_offered_conferences, Team.new()) }
+        end
+
+        context 'if it is a supervisor of another team' do
+           before do
+            allow(user).to receive(:supervisor?).and_return(true)
+          end
+          it { expect(ability).not_to be_able_to(:see_offered_conferences, Team.new()) }
+        end
+
+        context 'if it is a team\'s student' do
```

Comment:
Please use double quotes and don't escape the single quote then _(also in the other examples of course)_